mizar/dp/mizar/workflows/builtins/networkpolicies/create.py

```diff
@@ -103,3 +113,51 @@ def init_data_for_networkpolicy(self):
             "port_table": [],
         }
         return data
+
+    def fill_data_from_onegress(self, data, gress_type, networkpolicy):
```

Comment:
Same as above. you can name it something like access rules or traffic rules.

Reply:
Fixed